common/lib/driver-definitions/api-report/driver-definitions.api.md

```diff
@@ -224,6 +224,14 @@ export interface IGenericNetworkError extends IDriverErrorBase {
     readonly statusCode?: number;
 }
 
+// @public
+export interface ILoaderInfoProps {
```

Comment:
Isn't it a tooling bug that it shows up in api.md when you added a comment in .ts file? I.e. I'd expect it to be here, but with "(undocumented)" title before your change

Reply:
It is putting (undocumented) in case I don't have comment on top of it. If I put a comment, then it does not put (undocumented), which is the right behavior imo.